pkgs/development/libraries/qt-5/modules/qtbase.nix

```diff
@@ -381,4 +410,12 @@ stdenv.mkDerivation (finalAttrs: {
     platforms = platforms.unix;
   };
 
+}))
+.overrideAttrs(previousAttrs: lib.optionalAttrs (stdenv.buildPlatform != stdenv.hostPlatform) {
+  configurePlatforms = [ ];
+
+  # `qtbase` expects to find `cc` (with no prefix) in the
+  # `$PATH`, so the following is needed even if
+  # `stdenv.buildPlatform.canExecute stdenv.hostPlatform`
+  depsBuildBuild = (previousAttrs.depsBuildBuild or []) ++ [ buildPackages.stdenv.cc ];
 })
```

Comment:
Shouldn't need to be an overrideAttrs, a few `optionals` should work

Reply:
There is no `depsBuildBuild` in the original expression, so adding `depsBuildBuild=[]` to the native side will trigger rebuilds.

I expect that `staging` will merge to `master` before this PR is merged.  If that happens and I can avoid rebuilds, I will change the branch of this PR from `staging` to `master`.

Reply:
This PR now causes no rebuilds.